.changeset/slow-buses-rhyme.md

```diff
@@ -0,0 +1,5 @@
+---
+"@logto/cli": minor
+---
+
+add new cli command to setup proxy for developing and debugging custom ui locally
```

Comment:
should have some description about this?

Reply:
Updated